api.js

```diff
@@ -1536,6 +1537,65 @@ API.prototype.getTraceMetadata = function getTraceMetadata() {
   return metadata
 }
 
+/**
+ * Get a set of tracked identifiers
+ *
+ * @param {object} params Input parameters.
+ * @param {string} params.responseId The LLM generated identifier for the
+ * response.
+ * @returns {LlmTrackedIds|undefined} The tracked identifiers.
+ */
+API.prototype.getLlmTrackedIds = function getLlmMessageIds({ responseId } = {}) {
```

Comment:
```suggestion
API.prototype.getLlmMessageIds = function getLlmMessageIds({ responseId } = {}) {
```

This name needs updated to align with CDD and agreed upon method with python agent

Reply:
For what it's worth, I changed the name because the result doesn't really match the name 🤷‍♂️